src/components/MerchantPayout/AllValidVouchersModal.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FunctionComponent, useEffect } from "react";
+import {
+  StyleSheet,
+  View,
+  Modal,
+  TouchableWithoutFeedback,
+  TouchableOpacity,
+  Alert
+} from "react-native";
+import { size, color } from "../../common/styles";
+import { Card } from "../Layout/Card";
+import { Voucher } from "./MerchantPayoutScreen";
+import { ValidVoucherCount } from "./ValidVoucherCount";
+import { Feather } from "@expo/vector-icons";
+import { AppText } from "../Layout/AppText";
+import { ScrollView } from "react-native-gesture-handler";
+
+const styles = StyleSheet.create({
+  background: {
+    ...StyleSheet.absoluteFillObject,
+    backgroundColor: color("grey", 100),
+    opacity: 0.8
+  },
+  cardWrapper: {
+    padding: size(3),
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "center"
```

Comment:
I think don't need this, then `counterWrapper` can use `alignSelf: "flex-start"`

Reply:
This is required to center the card on the screen, cuz the modal just causes a popup! but ill change the alignSelf part!